frontend-web/webclient/app/Applications/Jobs/View.tsx

```diff
@@ -723,22 +743,37 @@ function isSupported(jobBackend: string | undefined, support: ComputeSupport | u
     }
 }
 
-const RunningText: React.FunctionComponent<{job: Job; interfaceLinks: string[]}> = ({job, interfaceLinks}) => {
-    return <>
-        <Flex justifyContent={"space-between"} height={"var(--logoSize)"}>
-            <Flex flexDirection={"column"}>
-                <Heading.h2>
-                    {job.specification?.name ?? job.status.resolvedApplication?.metadata?.title ?? "Your job"} is now running
-                    {" "}
-                    <Box style={{display: "inline"}} color={"textSecondary"}>(ID: {job.id})</Box>
-                </Heading.h2>
-                <Box flexGrow={1} />
-                <div><CancelButton job={job} state={"RUNNING"} /></div>
-            </Flex>
-            <RunningButtonGroup job={job} rank={0} redirectToWeb={interfaceLinks[0]} />
-        </Flex>
-    </>;
-};
```

Comment:
Why has this been removed instead of being in a separate component?

Reply:
I don't remember exactly, but it might have been an attempt to simplify/flatten the hierarchy a bit. I'll revert it back though.